{
  "name": "dataproc",
  "name_pretty": "Google Cloud Dataproc",
  "product_documentation": "https://cloud.google.com/dataproc",
  "client_documentation": "https://googleapis.dev/python/dataproc/latest",
  "issue_tracker": "https://issuetracker.google.com/savedsearches/559745",
  "release_level": "alpha",
  "language": "python",
  "repo": "googleapis/python-dataproc",
  "distribution_name": "google-cloud-dataproc",
  "api_id": "dataproc.googleapis.com",
  "requires_billing": true
}